### DISCUSSION TITLE
undici with socks5 proxy

### DISCUSSION THREAD

**Discussion by qualitydude** [nodejs/undici#1569] [Q&A]
Hi,

I was trying that in many ways, there is possibility to use **undici** with proxy server using SOCKS5 protocol?

I saw there is ProxyAgent for http but nothing for socks5. Can somone give me informations around it?

**Comment by mcollina** [comment]
There is no support for SOCKS5 atm. Somebody would need to implement SOCKS5 support. Would you?

**Comment by qualitydude** [comment]
I would love to see that implementation, but unfortunately I cannot do that :(
